crates/common_lang_types/src/string_types.rs

```diff
@@ -9,6 +11,6 @@ macro_rules! derive_display {
     };
 }
 
-#[derive(Debug)]
+#[derive(Debug, Serialize)]
```

Comment:
heh, we should probably not have `QueryText` and instead just have `OperationText`: #473 (no need to fix it here, unless you want to)